xlgui/widgets/playlist.py

```diff
@@ -1249,6 +1249,21 @@ def on_drag_data_received(self, widget, context, x, y, selection, info, etime):
         else:
             insert_position = -1
 
+        if self._hack_is_osx:
```

Comment:
Let's add a link to the original bug so someone in the future knows why this is here.

Reply:
I thought that git blame do that.